doc/usage.md

```diff
@@ -11,7 +11,7 @@ and [latest Electrum wallet](https://electrum.org/#download) (3.3+).
 Also, install the following packages (on Debian):
 ```bash
 $ sudo apt update
-$ sudo apt install clang cmake  # for building 'rust-rocksdb'
+$ sudo apt install clang cmake build-essential  # for building 'rust-rocksdb'
```

Comment:
I checked if `build-essential` adds a lot of bloat on top of what's required. It looks like it doesn't, so it's fine. :)